HW_3/task1.py

```diff
@@ -0,0 +1,19 @@
+# Задача 16: Требуется вычислить, сколько раз встречается некоторое число Х
+# в массиве А[1..N]. Пользователь в первой строке вводит натуральное число N -
+# количество элементов в массиве. В последующих строках записаны N целых
+# чисел Ai. Последняя строка содержит число x
+
+
+print('Введите натуральное число N') 
+N = abs(int(input()))
```

Comment:
Без функции `print` здесь можно вполне обойтись.
```suggestion
N = abs(int(input('Введите натуральное число N '))
```